Euclidean_Algorithm/Euclidean_Algorithm.c

```diff
@@ -0,0 +1,31 @@
+// Program to demonstrate Basic Euclidean Algorithm 
+#include <stdio.h> 
+
+// Function to return gcd of x and y
+int gcd_algorithm(int x,int y)
+{
+   // if the remainder is 0, return second number
```

Comment:
This comment needs to be indented one more space so that it falls in line with code. Please add one space here.